content/24.vaccines-novel.md

```diff
@@ -431,52 +430,51 @@ Similar to BNT162b2, the mRNA-1273 vaccine was associated with mild-to-moderate
 
 The WHO continues to monitor the emergence of variants and their impact on vaccine efficacy [@url:https://www.who.int/news-room/feature-stories/detail/the-effects-of-virus-variants-on-covid-19-vaccines].
-Previous research in the computational prediction of the efficacy of vaccines targeting the influenza A virus might complement efforts to monitor these types of viral outbreaks [@doi:10.1093/cid/ciy323].
-To adapt, booster shots are now recommended in many places, and vaccines in the near future will likely account for multiple variants and strains of SARS-CoV-2 [@doi:10.1038/s41591-021-01230-y].
+In general, mRNA vaccines remain highly effective against severe illness and death, but the effectiveness against infection generally has declined.
+A study monitoring infections in a Minnesota cohort from January to July 2021 estimated that the effectiveness of the Moderna vaccine fell to 86% and Pfizer to 76%, although protection against hospitalization remained at 91% and 85%, respectively [@doi:10.1101/2021.08.06.21261707].
+In July of that year, as the Delta variant became dominant in the U.S.A., these estimates all fell, to an effectiveness of 76% for Moderna and 42% for Pfizer and effectiveness against hospitalization of 81% and 75%, respectively [@doi:10.1101/2021.08.06.21261707].
+
+With the emergence of the Omicron VOC, vaccine effectiveness has likely further declined.
+A study in a diverse cohort in Southern California, U.S.A. found the effectiveness of the Moderna vaccine in participants who had received only the primary course to be 44% [@doi:10.1038/s41591-022-01753-y].
+A study in South Africa compared case and hospitalization records from a 4-week period where Omicron was dominant to a 2-month period where Delta was dominant and found that the effectiveness against hospitalization during the Omicron wave was approximately 70% compared to 93% during the Delta wave [@doi:10.1056/NEJMc2119270].
+Similarly, a large study in England of 2.5 million individuals suggested that not only the variants circulating, but also the time since vaccination, played a large role in vaccine effectiveness [@doi:10.1056/NEJMoa2119451].
+Shortly after the BNT162b2 primary course, effectiveness against the Omicron VOC was as high as 65.5%, but this declined to below 10% by six months after the second dose.
+For mRNA-1273, the decline was from 75.1% to 14.9%.
+Therefore, it is unsurprising that in spite of vaccination programs, infection rates and hospitalization rates climbed in early 2022 in many Western countries including the United States [@url:https://www.nytimes.com/interactive/2021/us/covid-cases.html; @{url:https://www.washingtonpost.com/health/interactive/2022/omicron-comparison-cases-deaths-hospitalizations/}].
+
+On the side of safety, the only major concern that has been raised is a possible link between mRNA vaccination and myocarditis, especially in young men [@doi:10.15585/mmwr.mm7032e4].
```

Comment:
Added this paragraph in response to feedback from Tony

Reply:
Maybe a drop in non-pharmacological approaches to mitigation were a factor too for early 2022

Reply:
see above!

Reply:
Thanks!